src/classes/Customer.js

```diff
@@ -0,0 +1,94 @@
+import dayjs from 'dayjs';
+import isBetween from 'dayjs/plugin/isBetween';
+
+dayjs.extend(isBetween)
+
+class Customer {
+  constructor(customer) {
+    this.id = customer.id;
+    this.name = customer.name;
+    this.bookings = [];
+  }
+
+  getBookings(bookings) {
```

Comment:
Should I have a method to addNewBookings Or should I call this method again after a new bookings have been posted?